circle.yml

```diff
@@ -65,3 +82,12 @@ test:
             background: true
         - sleep 5
         - mocha-phantomjs http://localhost:5000/base/test/index.html
+
+        # Docker
+        - docker run -d --name db postgresql
+        - docker run -d --name solr solr
+        - docker run -d --name redis redis:latest
+        - docker run -d --name ckan -p 5001:5000 --link db:db --link redis:redis --link solr:solr ckan
+        - sleep 10
+        - curl --retry 10 --retry-delay 5 -v http://localhost:5001
+
```

Comment:
The circlce-matrix docs are more than a little lacking, let me know if you'd like more direction here